graph/src/blockchain/mod.rs

```diff
@@ -79,29 +64,39 @@ pub trait Blockchain: Sized + Send + Sync + 'static {
     /// Trigger filter used as input to the triggers adapter.
     type TriggerFilter: TriggerFilter<Self>;
 
-    type NodeCapabilities;
+    type NodeCapabilities: std::fmt::Display;
 
     type IngestorAdapter: IngestorAdapter<Self>;
 
     // type RuntimeAdapter: RuntimeAdapter;
     // ...WIP
 
     fn reorg_threshold() -> u32;
```

Comment:
We can just remove this, it's not needed here.

Reply:
Sadly, it is as an argument to `Blockchain.triggers_adapter`.